.github/workflows/deploy-dev-pkgs.yml

```
@@ -0,0 +1,82 @@
+name: Deploy "conda" conda packages
+
+on:
+  workflow_run:
+    workflow:
+      - CI tests
+    branches:
+      - master
+    types:
+      - completed
+
+jobs:
+  # Check for required configuration/secrets needed to build & upload.  Done
+  # this way because secrets cannot be used directly in GitHub Action
+  # conditionals; see https://github.com/actions/runner/issues/520 for details.
+  check-config:
+    runs-on: ubuntu-18.04
+    outputs:
+      okay: ${{ steps.check-secrets.outputs.defined }}
+    steps:
+      - name: Check for secrets needed to build & upload packages
+        id: check-secrets
+        env:
+          ANACONDA_ORG_CHANNEL: ${{ secrets.ANACONDA_ORG_CHANNEL }}
+          ANACONDA_ORG_LABEL: ${{ secrets.ANACONDA_ORG_LABEL }}
+          ANACONDA_ORG_TOKEN: ${{ secrets.ANACONDA_ORG_TOKEN }}
+        shell: bash -l {0}
+        run: |
+          test -n "${ANACONDA_ORG_CHANNEL}" || \
+              { echo "FATAL: ANACONDA_ORG_CHANNEL secret not configured" >&2; exit 1; }
+          test -n "${ANACONDA_ORG_LABEL}" || \
+              { echo "FATAL: ANACONDA_ORG_LABEL secret not configured" >&2; exit 1; }
+          test -n "${ANACONDA_ORG_TOKEN}" || \
+              { echo "FATAL: ANACONDA_ORG_TOKEN secret not configured" >&2; exit 1; }
+          echo "::set-output name=defined::true"
+
+  build-conda-pkgs:
+    needs:
+      - check-config
+    if: |
+      github.event.workflow_run.conclusion == 'success' &&
+      needs.check-config.outputs.okay == 'true'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-18.04
+            subdir: linux-64
+          - runner: macos-10.15
+            subdir: osx-64
+          - runner: windows-2019
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+      - uses: conda-incubator/setup-miniconda@v2
+      - name: Build & upload package
+        env:
+          ANACONDA_ORG_TOKEN: ${{ secrets.ANACONDA_ORG_TOKEN }}
+
+          # These don't really need to be secrets, but we don't really have
+          # another way of performing per-repository/-fork configuration.
+          #ANACONDA_ORG_CHANNEL: ${{ secrets.ANACONDA_ORG_CHANNEL }}
+          #ANACONDA_ORG_LABEL: ${{ secrets.ANACONDA_ORG_LABEL }}
+        shell: bash -l {0}
+        run: |
+          set -eu
+          conda activate
+          conda update --yes conda
+          conda install --yes conda-build anaconda-client git
+          conda info
+          conda config --show-sources
+          conda list
+          conda build --croot=./pkgs conda.recipe
+          anaconda --token ${ANACONDA_ORG_TOKEN} upload \
```

Since this is a secrete can we just leave it as such and then remove the `env` defined above and move the above comment down here?
```suggestion
          anaconda --token ${{ secrets.ANACONDA_ORG_TOKEN }} upload \
```

---

Done; the GHA docs are a bit ambiguous about whether the `env` block is strictly needed to properly handle secrets, but logs from the test runs seem to confirm that it's not necessary; i.e., directly passing the token as suggested isn't causing it to show up in the logs.